src/redux/financialStatements/finStatements.js

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+
+const baseUrl = 'https://api.binance.com/api/v3/ticker/24hr';
+const DISPLAY_STOCKS = 'DISPLAY_STOCKS';
+const STOCK_DETAILS = 'STOCK_DETAILS';
```

Comment:
- [OPTIONAL] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:  
`const STOCK_DETAILS = .../financialStatements/STOCK_DETAILS `